quickstart/k8s-the-prod-way/identity.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Bootstrapping Identity
+---
+
+Begin by cloning the [Kubernetes the Prod Way repository][ktpw]. The `identity` stack is implemented
+for each of [AWS][aws], [Azure][azure], and [GCP][gcp].
+
+This stack is the root from which we will bootstrap the rest of our infrastructure. In this
+tutorial, we will:
+
+* Provision service accounts for CI/CD
+* Provision roles for users and teams
+
+**IMPORTANT CAVEAT:** The AWS IAM role and Azure AD account APIs are not do not yet fully exposed
+through the Pulumi SDK. This stack will provision them, but you will have to manually grant them
+permissions.
+
+## Prerequisites: Bootstrapping Identity
+
+In order to provision identities:
+
+1. Someone must create an admin account in the relevant cloud provider (AWS, Azure, or GCP)
+1. That admin account must provision the initial set of identities, so that CI/CD and teams can
+   provision and manage infrastructure.
+
+Before you begin, you'll need to do this and then log in through the CLI:
+
+* For AWS, run `aws configure` (see [docs][aws-cli])
+* For Azure, run `az login` (see [docs][az-cli])
+* For GCP, run `gcloud auth` (see [docs][gcp-cli])
+
+## Provisioning
+
+Once you've logged in through the CLI, `cd` into the `identity` directory of the cloud provider
```

Comment:
Instead of making these directories, we could use config to make it a deployment time decision.

Reply:
(this often makes it much easier to share code)

Reply:
We could, but that's not going to make it this morning. We will have to do it later.